tests/test_printer.py

```diff
@@ -494,13 +504,14 @@ class CustomFormatAttr(ParametrizedAttribute):
     attr: ParameterDef[IntAttr]
 
     @staticmethod
-    def parse_parameters(parser: Parser) -> list[Attribute]:
+    def parse_parameters(parser: BaseParser) -> list[Attribute]:
         parser.parse_char("<")
-        value = parser.parse_alpha_num(skip_white_space=False)
-        if value == "zero":
+        value = parser.tokenizer.next_token_of_pattern(
```

Comment:
I would argue that writing something like this would be more "intuitive" from a user perspective.
I feel that using regexes here is a bit overkill?
```
if parser.parse_optional_keyword("zero") is not None:
  ...
  return ...
parser.parse_keyword("one")
...
```